src/simulation.hpp

```diff
@@ -981,7 +1021,20 @@ template <typename problem_t> void AMRSimulation<problem_t>::calculateGpotAllLev
 			rhs[lev].define(grids[lev], dmap[lev], ncomp, nghost);
 			phi[lev].setVal(0); // set initial guess to zero
 			rhs[lev].setVal(0);
+		}
+
+#ifdef AMREX_PARTICLES
+		if (do_cic_particles != 0) {
+			// deposit particles using amrex::ParticleToMesh
+			amrex::ParticleToMesh(*CICParticles, amrex::GetVecOfPtrs(rhs), 0, finest_level,
+					      quokka::CICDeposition{Gconst_, quokka::ParticleMassIdx, 0, 1});
```

Comment:
Unclear whether `amrex::ParticleToMesh` or `CICParticles->AssignDensity` is the appropriate function to use here.

Reply:
`AssignDensity` only does CIC deposition, whereas `ParticleToMesh` is more general. Either should work.